x-pack/plugins/osquery/public/shared_components/osquery_action/use_osquery_menu_item.tsx

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
this component should live inside `security_solution` in my opinion, because it will be only used in alert details

Reply:
Moved 👍